### ISSUE TITLE
Abnormal Train Loss 

### ISSUE THREAD

**Issue by simmonssong** [liyaguang/DCRNN#38]
Hi, thanks for your code.
I got an abnormal training loss under default config. Although validating error is convergent, training loss goes down at first but it **keeps growing** after minimal. 
* METR-LA
![image](https://user-images.githubusercontent.com/15140733/61995840-04aa7480-b0c0-11e9-88a5-779121512852.png)
* PEMS-BAY
![image](https://user-images.githubusercontent.com/15140733/61995684-33bfe680-b0be-11e9-97ac-a83057b98cd1.png)

**Comment by liyaguang** [comment]
This behavior is expected due to the scheduled sampling. In this process, the training task keeps getting hard, and finally converges to the testing/validation task. You may refer to Section 2.3 in the [paper](https://arxiv.org/pdf/1707.01926.pdf) for more information.

**Comment by simmonssong** [comment]
> This behavior is expected due to the scheduled sampling. In this process, the training task keeps getting hard, and finally converges to the testing/validation task. You may refer to Section 2.3 in the paper for more information.

I got it. Thanks a lot.
